README.md

```diff
@@ -140,56 +193,50 @@ By default, Sheer will render pages at their natural paths in the project's file
 structure.
 For example, going to <http://localhost:7000/the-bureau/index.html>
 (or <http://localhost:7000/the-bureau/>) renders `/the-bureau/index.html`
-as processed by the [Jinja2](http://jinja.pocoo.org/docs/) templating engine.
-Note that this page does not automatically show any content indexed by Sheer;
+as processed by the [Jinja2](http://jinja.pocoo.org/docs) templating engine.
+**NOTE:** This page does not automatically show any content indexed by Sheer;
 it simply outputs the static HTML written into the template.
 
 ### Outputting indexed content in a Sheer template
 
-Most of our content is indexed from the API output of our WordPress back end. This happens when the `sheer index` command is run.
-(WordPress was previously used to serve the site front-end,
-but going forward, it will only be used as a content editing and storage system.)
-
-If your content isn't being indexed yet, see "Setting up a new WordPress post
-type and processing it with Sheer" on the flapjack/Getting-started-with-Flapjack
-wiki (on our GitHub Enterprise server).
+Most of our content is indexed from the API output of our WordPress back-end. This happens when the `sheer index` command is run.
 
-There are two ways in which we use indexed content: repeating items
-(e.g., blog posts and press releases), and single pages
-(e.g., the Future Requests page in Doing Business with Us).
+There are two ways in which we use indexed content: repeating items (e.g., blog posts and press releases), and single pages (e.g., the Future Requests page in Doing Business with Us). What follows is a deeper dive into both of these content types.
 
 #### Repeating content
 
 For any kind of repeating content, this is the basic process:
 
 1. In the vars file for the section you're in (e.g., `blog/_vars-blog.html`), we
-   set up a variable that holds the results of the default query we want to run.
+  set up a variable that holds the results of the default query we want to run.
+
+  Here's how it looks for the blog:
 
-   Here's how it looks for the blog:
+  ```jinja
```

Comment:
I was a little unclear what was going on here... the whole working templates section had what looked like character artifacts(?), removing them doesn't appear to affect how the markdown renders.